lib/src/model/data.dart

```diff
@@ -21,7 +21,7 @@ import './review.dart';
 // This is the file that Codelab users will primarily work on.
 
 Future<void> addRestaurant(Restaurant restaurant) {
-  final restaurants = Firestore.instance.collection('restaurants');
+  final restaurants = FirebaseFirestore.instance.collection('restaurants');
```

Comment:
These need to be added in the actual document of the codelab, I'll get those in.

Reply:
This is under review right now.